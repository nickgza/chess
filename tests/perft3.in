fen
8/2p5/3p4/KP5r/1R3p1k/8/4P1P1/8 w - -
perft
1
2
3
4
5
q
