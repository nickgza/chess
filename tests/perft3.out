14
191
2812
43238
674624
